src/_internals/item/ItemRadioCheckbox.style.tsx

```diff
@@ -0,0 +1,24 @@
+import styled from 'styled-components'
+
+import { color, inputBorderSize } from '../../_utils/branding'
+import { Item } from './index'
+
+export const StyledItemRadioCheckbox = styled(Item)`
```

Comment:
Not sure, by an (implicit) convention in our codebase we prefix all our components wrapped with `styled` by `Styled*`

Reply:
I though we prefixed them only to avoid name conflict with the exported component using the styled component.
I find it weird having an exported `StyledItemRadioCheckbox` from a _ItemRadioCheckbox.style.tsx_ file, it feels like we're exposing implementation details.

Reply:
You're right, I renamed it to `ItemRadioCheckbox`